packages/remix-vercel/package.json

```
@@ -16,12 +16,13 @@
   },
   "devDependencies": {
     "@types/supertest": "^2.0.10",
-    "@vercel/node": "^1.8.3",
+    "@vercel/node": "^2.0.1",
+    "@vercel/node-bridge": "^3.0.0",
     "node-mocks-http": "^1.10.1",
     "supertest": "^6.1.5"
   },
   "peerDependencies": {
-    "@vercel/node": "^1.8.3"
+    "@vercel/node": "^1.8.3 || ^2.0.0"
```

Not sure about this tbh 🤔
Technically `2.0` if fine if people install `@vercel/build-utils` themselves

```suggestion
    "@vercel/node": "^1.8.3 || ^2.2.0"
```

---

i think 2.2 is fine as it doesn't require the extra dep in user land